react/features/settings/components/native/SettingsView.tsx

```diff
@@ -463,29 +560,53 @@ class SettingsView extends AbstractSettingsView<Props, State> {
     }
 
     /**
-     * Callback to update the start with audio muted value.
+     * Handles the start audio muted change event.
      *
-     * @param {boolean} startWithAudioMuted - The new value to set.
+     * @param {boolean} startWithAudioMuted - The new value for the start audio muted
+     * option.
+     * @protected
      * @returns {void}
      */
-    _onStartAudioMutedChange(startWithAudioMuted) {
-        super._onStartAudioMutedChange(startWithAudioMuted);
+    _onStartAudioMutedChange(startWithAudioMuted: boolean) {
         this.setState({
             startWithAudioMuted
         });
+
+        this._updateSettings({
+            startWithAudioMuted
+        });
     }
 
     /**
-     * Callback to update the start with video muted value.
+     * Handles the start video muted change event.
      *
-     * @param {boolean} startWithVideoMuted - The new value to set.
+     * @param {boolean} startWithVideoMuted - The new value for the start video muted
+     * option.
+     * @protected
      * @returns {void}
      */
-    _onStartVideoMutedChange(startWithVideoMuted) {
-        super._onStartVideoMutedChange(startWithVideoMuted);
+    _onStartVideoMutedChange(startWithVideoMuted: boolean) {
         this.setState({
             startWithVideoMuted
         });
+
+        this._updateSettings({
+            startWithVideoMuted
+        });
+    }
+
+    /**
+     * Handles the start reactions muted change event.
+     *
+     * @param {boolean} startWithReactionsMuted - The new value for the start reactions muted
+     * option.
+     * @protected
+     * @returns {void}
+     */
+    _onStartReactionsMutedChange(startWithReactionsMuted: boolean) {
```

Comment:
There seems to be no control which uses this. Does it make sense to add this now? Or can we defer it until we refactor the settings screen?

Reply:
I can remove it yes. No use for it now.